SmartDeviceLink/SDLStreamingAudioLifecycleManager.m

```diff
@@ -190,10 +190,10 @@ - (void)didEnterStateAudioStreamShuttingDown {
     [self.protocol endServiceWithType:SDLServiceTypeAudio];
 }
 
-#pragma mark - SDLProtocolListener
+#pragma mark - SDLProtocolDelegate
 #pragma mark Start Service ACK/NAK
 
-- (void)handleProtocolStartServiceACKMessage:(SDLProtocolMessage *)startServiceACK {
+- (void)protocol:(SDLProtocol *)protocol didReceiveStartServiceACK:(SDLProtocolMessage *)startServiceACK {
```

Comment:
Update the debug log in here with the transport info?

Reply:
Added.